user_management/api/tests/test_views.py

```diff
@@ -62,9 +62,8 @@ def test_post(self):
     def test_post_last_login_updates(self):
         """Authenticating updates the user's last_login."""
         user = UserFactory.create(email=self.username, password=self.password)
-        now = timezone.now()
-        self.assertLess(user.last_login, now)
```

Comment:
The other possibility is to explicitly set `last_login` to a suitable datetime when creating the `user`.